.github/workflows/test.yml

```diff
@@ -170,7 +171,170 @@ jobs:
 
         # Start mongo container and app before running api tests
         run: |
-          cp CI/ESS/docker-compose.api.yaml docker-compose.yaml
           cp functionalAccounts.json.test functionalAccounts.json
-          docker-compose up --build -d
-          npm run test:api
+          docker-compose up --build -d -f CI/ESS/docker-compose.api.yaml
```

Comment:
wrong  docker-compose syntax

` docker-compose  -f CI/ESS/docker-compose.api.yaml up --build -d`

(`-f` is a global flag, `--build` and `-d` are flags for up)